src/components/DataTable/DataTable.js

```diff
@@ -162,14 +164,16 @@ class DataTable extends Component {
   }
 
   renderHead() {
+    const { fixedHeader } = this.props;
     const { columns, id, sortBy, sortDirection } = this.state;
 
     return (
       <thead>
-        <tr>
+        <tr className={fixedHeader ? 'slds-text-title_caps' : null}>
```

Comment:
Why does `fixedHeader` imply text in caps?

Reply:
Just took the markup from the SF docs for a fixed header. Maybe we can remove this.